core/src/avm1/globals/file_reference_list.rs

```diff
@@ -0,0 +1,141 @@
+use crate::avm1::activation::Activation;
+use crate::avm1::error::Error;
+use crate::avm1::function::FunctionObject;
+use crate::avm1::globals::as_broadcaster::BroadcasterFunctions;
+use crate::avm1::globals::file_reference::FileReferenceObject;
+use crate::avm1::property_decl::{define_properties_on, Declaration};
+use crate::avm1::{ArrayObject, Executable, NativeObject, Object, ScriptObject, TObject, Value};
+use crate::backend::ui::FileFilter;
+use crate::context::GcContext;
+use gc_arena::GcCell;
+
+const PROTO_DECLS: &[Declaration] = declare_properties! {
+    "browse" => method(browse; DONT_ENUM);
+};
+
+const OBJECT_DECLS: &[Declaration] = declare_properties! {};
```

Comment:
I guess a note here about why `fileList` is not present here would be nice, but not that important.

Reply:
Also, why set it manually in the constructor instead of adding it here? I'm not criticizing, just curious.